### PR TITLE
feat: amount input, SendScreen cleanup

### PR DESCRIPTION
**Pull request by dcposch**
## Better amount entry, send flow

- [x] Simplify SendScreen
- [x] Refactor AmountInput, remove toggle between input and label
- [x] User should never have to dismiss the keyboard to proceed
- [x] Fix CreateNoteTab
- [x] Layout

image

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/send/SendNoteTab.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { View } from "react-native";
+
+import { SendNoteButton } from "./SendNoteButton";
+import { useAvailMessagingApps } from "../../../logic/messagingApps";
+import { SendAmountChooser } from "../../shared/AmountInput";
+import Spacer from "../../shared/Spacer";
+import { TextCenter, TextLight } from "../../shared/text";
+
+export function SendNoteTab() {
+  // Create Note shows available secure messaging apps
```
Comment (nalinbhardwaj) [inline apps/daimo-mobile/src/view/screen/send/SendNoteTab.tsx:11]: nit: `Create` -> `Send`?

File: apps/daimo-mobile/src/view/screen/send/SendScreen.tsx [hunk 2]
```diff
@@ -50,123 +27,75 @@ export default function SendScreen({ route }: Props) {
...
+        values={tabs}
+        selectedIndex={tabs.indexOf(tab)}
+        onValueChange={setTab}
+        fontStyle={{ fontSize: 16 }}
+        activeFontStyle={{ fontSize: 16 }}
+        style={{ height: 40 }}
+      />
+      <Spacer h={8} />
+      {tab === "Search" && <SearchTab />}
+      {tab === "Send Link" && <SendNoteTab />}
```
Comment (nalinbhardwaj) [inline apps/daimo-mobile/src/view/screen/send/SendScreen.tsx:63]: should there be a `hide` or similar param here too like scan?

**Review by nalinbhardwaj — APPROVED** [review]
code looks great so far, should test android too

File: apps/daimo-mobile/src/view/screen/send/SendNoteTab.tsx (hunk 1 above)
Comment (dcposch) [reply apps/daimo-mobile/src/view/screen/send/SendNoteTab.tsx:11]: for sure.

i might rename everything to PaymentLink (the contract can still be `EphemeralNotes`, but won't use "notes" anywhere else)

File: apps/daimo-mobile/src/view/screen/send/SendScreen.tsx (hunk 2 above)
Comment (dcposch) [reply apps/daimo-mobile/src/view/screen/send/SendScreen.tsx:72]: fixed, removed from ScanTab

File: apps/daimo-mobile/src/view/screen/send/SendNoteTab.tsx (hunk 1 above)
Comment (dcposch) [reply apps/daimo-mobile/src/view/screen/send/SendNoteTab.tsx]: fixed

**Pull request merged**